README.md

```diff
@@ -54,6 +54,9 @@ worker.end(true);
 try { workerThread.join(); } catch (Exception e){ e.printStackTrace(); }
 ```
 
+If enqueueing multiple jobs at the same time, there is `client.batchEnqueue(String queue, List<Job> jobs)` which does it
```

Comment:
should we maybe mention how does it do it and why is it more optimal?

Reply:
I think this is too detailed for the README. The section is called 'Quickstart'.